### ISSUE TITLE
What is the argument "holdout_size"?

### ISSUE THREAD

**Issue by ZebinYang** [microsoft/interpret#80]
In the Python API, ExplainableBoostingRegressor has two arguments named "holdout_split" and "holdout_size". However, after checking the source codes, I found "holdout_size" is not used in "ebm.py". May I know what is the difference between these two arguments?

**Comment by interpret-ml** [comment]
Hi @ZebinYang, 

```holdout_split``` is the parameter you want to use, and is passed directly into scikit-learn's [```train_test_split```](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html ) function internally. We use this to create an internal validation set for things like early stopping.

You're absolutely correct -- ```holdout_size``` is not currently used publicly, and should be avoided. We're going to make argument changes and clean this up in a future release of interpret, but because it is a breaking change, we'll give plenty of notice and an appropriate version number bump. 

-InterpretML Team
